File: _test-stacks/packaging-e2e/src/php-lambda/handler.php
<?php

require_once __DIR__ . '/vendor/autoload.php';

use Monolog\Logger;

function handler($event, $context)
{
    // Use Monolog to verify dependency was packaged
    $logger = new Logger('test');
    return [
        "statusCode" => 200,
        "body" => "php ok - monolog v" . Logger::API
    ];
}

File: _test-stacks/packaging-e2e/src/rb-lambda/handler.rb
require 'active_support'
require 'active_support/core_ext/string'

def handler(event:, context:)
  # Use ActiveSupport to verify dependency was packaged
  test_string = "hello world".titleize
  { statusCode: 200, body: "rb ok - #{test_string}" }
end
